src/Workspaces/Core/Portable/Workspace/Host/Mef/MefHostServices.cs

```diff
@@ -99,11 +99,15 @@ public static ImmutableArray<Assembly> DefaultAssemblies
         private static ImmutableArray<Assembly> LoadDefaultAssemblies()
         {
             // build a MEF composition using the main workspaces assemblies and the known VisualBasic/CSharp workspace assemblies.
+            // updated: includes feature assemblies since they now have public API's.
```

Comment:
I'd also prefer if we just updated the comment to state the current world rather than "updated, the previous line is now lying to you" comments.